### PR TITLE
Fix RLC values in Copy/Keccak table lookup (SHA3)

### PR DESCRIPTION
**Pull request by roynalnaruto**
Fixes the `integration-tests` warning after merging #635 for `SHA3` using a dummy gadget.

Still investigating failure because of Keccak lookup.

### CODE REVIEW

**Review by icemelon — APPROVED** [review]
LGTM

**Comment by lispc** [comment]
this pr looks good to me. But what about also fixing integration test in this PR?

**Comment by roynalnaruto** [comment]
> But what about also fixing integration test in this PR?

Adding a commit now to fix that!

**Comment by roynalnaruto** [comment]
Because of the Keccak table assignment update, the bytecode tests are failing. Will fix the bytecode config assignment as well.

File: zkevm-circuits/Cargo.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ sha3 = "0.10"
 digest = "0.7.6"
 array-init = "2.0.0"
 paste = "1.0"
-bus-mapping = { path = "../bus-mapping" }
+bus-mapping = { path = "../bus-mapping", features = ["test"] }
```
Comment (lispc) [inline zkevm-circuits/Cargo.toml:19]: It is possible to make zkevm circuits only need busmapping test feature when zkevm circuit is also in test mode？

Comment (roynalnaruto) [reply zkevm-circuits/Cargo.toml:19]: Yes I will make the change

**Review by lispc — APPROVED** [review]

**Pull request merged**